packages/select/docs/Select/Select.stickersheet.stories.tsx

```diff
@@ -39,9 +39,10 @@ const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
           items={singleMockItems}
           description="This is a description"
           selectedKey="id-sre"
-          trigger={(triggerProps): JSX.Element => (
+          trigger={(triggerProps, buttonRef): JSX.Element => (
```

Comment:
I'm not a fan of the consumer now being forced to add the ref. Take a look at the `Filter` component where the `ref` is cloned on behalf of the consumer.